src/services/EcoBridge/Socket/Socket.utils.ts

```diff
@@ -222,9 +222,9 @@ export const VERSION = {
   patch: 0,
 }
 
+// TODO: To be reverted later
```

Comment:
what needs to be reverted?

Reply:
the URL right below, that list was changed to point to the branch.